source/features/inline-code.js

```diff
@@ -34,9 +34,10 @@ export default function () {
 
 		const frag = text.map(val => {
 			// Style the single backticks while ignoring the already styled multiline code blocks
-			if (isElement(val)) {
+			if (isElement(val) || typeof val === "object") {
```

Comment:
Can you add a comment about this change? Not clear why it's needed. It should also use single-quotes.

Reply:
Oh sorry, I have commited a suggestion on pulling the navbar closer to the center but rebasing onto master didn't solve it. What am I doing wrong?

![image](https://user-images.githubusercontent.com/27804269/48021893-026f4700-e13a-11e8-8868-85876d85ebaf.png)

Regarding the change on inline-code.js, I removed that check. Previously for some reason the text array would contain elements that were objects and split would thus throw us an error but it seems to have disappeared now so it is removed